vistautils/range.py

```diff
@@ -1154,28 +1213,86 @@ def get_enclosed_by(self, rng: Range[K]) -> ImmutableSet[V]:
         raise NotImplementedError()
 
     @abstractmethod
-    def get_from_maximal_containing_or_below(self, key: K):
+    def get_from_rightmost_containing_or_below(self, key: K):
         """
-        Get the value associated with the maximal range in this set whose lower bound does not
+        Get the value associated with the rightmos range in this set whose lower bound does not
```

Comment:
Typo: rightmos -> rightmost
```suggestion
        Get the value associated with the rightmost range in this set whose lower bound does not
```

Reply:
Good catch, thanks!